README.md

```diff
@@ -117,6 +117,34 @@ When the pipeline runs against one of the provider-specific repos (e.g. cluster
 * The pipeline will use the latest tagged release of the cluster-test-suites container image.
 * Only the tests associated with that provider will be run, including the `upgrade` tests.
 
+### Running a specific subset of test suites
+
+It's possible to specify the test suites you'd like to run in CI by providing the `TARGET_SUITES` parameter with your comment trigger. This is useful when a test suite has failed due to what seems to be a flakey tests as you can re-run just the failing without wasting resources on the other test suites.
```

Comment:
```suggestion
It's possible to specify the test suites you'd like to run in CI by providing the `TARGET_SUITES` parameter with your comment trigger. This is useful when a test suite has failed due to what seems to be a flakey test as you can re-run just the failing without wasting resources on the other test suites.
```

Reply:
Doh! Sorry I had auto-merge on. I'll fix this in another commit